browser/brave_stats/buildflags.gni

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+declare_args() {
+  brave_stats_updater_url = ""
+  android_feedback_report_api_endpoint = ""
```

Comment:
this should use `GetServicesDomain` as well. We don't want to add more require urls configs